spring-integration-core/src/main/java/org/springframework/integration/gateway/GatewayProxyFactoryBean.java

```diff
@@ -311,7 +311,6 @@ public String call() throws Exception {
 					this.asyncSubmitListenableType = submitType.getClass();
 				}
 			}
-			this.start();
```

Comment:
Although this is wrong, I don't think we can make this change on the backport - it could break usage outside of a Spring application context.